inc/Extension/AuthPlugin.php

```diff
@@ -20,20 +20,31 @@ abstract class AuthPlugin extends Plugin
      * do. The things a backend can do need to be set to true
      * in the constructor.
      */
-    protected $cando = array(
-        'addUser' => false, // can Users be created?
-        'delUser' => false, // can Users be deleted?
-        'modLogin' => false, // can login names be changed?
-        'modPass' => false, // can passwords be changed?
-        'modName' => false, // can real names be changed?
-        'modMail' => false, // can emails be changed?
-        'modGroups' => false, // can groups be changed?
-        'getUsers' => false, // can a (filtered) list of users be retrieved?
-        'getUserCount' => false, // can the number of users be retrieved?
-        'getGroups' => false, // can a list of available groups be retrieved?
-        'external' => false, // does the module do external auth checking?
-        'logout' => true, // can the user logout again? (eg. not possible with HTTP auth)
-    );
+    protected $cando = [
+        'addUser' => false,
+        // can Users be created?
```

Comment:
Comments are placed below, if not next to it (because too long?), better above it. The last comment was dropped.

Reply:
This is what I meant with "rector isn't very good with white spaces". Since it uses an AST it isn't really aware of comments at all. I tried to fix issues like that manually when I noticed. Please feel free to push additional manual clean up commits.